python/practices/practice-3/src/path2/random_word_selector.py

```diff
@@ -0,0 +1,93 @@
+"""
+This program randomly selects a word from a given ebook. It cleans and
+processes the text, creating a histogram of word frequencies. Then, it selects
+a random word with a probability proportional to its frequency.
+"""
+
+
+import string
+import collections
+import os
+from random import randint
+from bisect import bisect
+from constants.constants import EBOOK
+from helpers.file_reader import read_file
+
+
+def clean_file(file_content):
```

Comment:
Please update the name of the function and param.

Reply:
Done